src/xunit.performance.api/PerformanceTestConfig.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public class PerformanceTestConfig
+    {
+        public PerformanceTestConfig() 
+        {
+            Iterations = 30;
+            TimeoutPerIteration = TimeSpan.FromMilliseconds(20000);
+        }
+
+        public TimeSpan TimeoutPerIteration 
+        {
+            get { return _timeoutPerIteration; }
+
+            set
+            {
+                if (value.TotalMilliseconds > 0) 
+                {
+                    _timeoutPerIteration = value;
+                }
+                else 
+                {
+                    throw new InvalidOperationException("Timeout per iteration should be a positive amount of milliseconds.");
+                }
+            }
+        }
+        public int Iterations
+        {
+            get { return _iterations; }
+
+            set
+            {
+                if (value > 1) 
```

Comment:
@brianrob How do `[Benchmark]` style tests treat iteration counts? Do you have to say `11` if you want `10` iterations, or does it automatically run one more?